autoware_utils/test/src/geometry/test_boost_polygon_utils.cpp

```diff
@@ -0,0 +1,331 @@
+// Copyright 2020 Tier IV, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "autoware_utils/geometry/boost_polygon_utils.hpp"
+#include "autoware_utils/geometry/geometry.hpp"
+
+#include <boost/geometry/geometry.hpp>
+
+#include <gtest/gtest.h>
+
+using autoware_utils::Polygon2d;
+
+namespace
+{
+geometry_msgs::msg::Point32 createPoint32(const double x, const double y)
```

Comment:
Given that the rest of the functions use snake case, can this be renamed to `create_point32`?